app/src/main/res/layout/activity_ownerbooks.xml

```diff
@@ -71,11 +73,12 @@
                 android:layout_width="150dp"
                 android:layout_height="wrap_content"
                 android:checked="true"
-                android:text="@string/lent" />
+                android:text="@string/borrowed" />
```

Comment:
I don't think this is supposed to change, the owner page has things you've, the borrower page has things you've borrowed